translation/en.json

```
@@ -0,0 +1,49 @@
+{
+    "number_topic":"Topic number",
+    "comment_posted":"Comment Posted ✅",
+    "first_link_your_account":"⛔ You need to link your account first",
+    "first_link_your_account_matrix":"⛔ You need to link your account, by sending 'discourse' or '6'",
+    "writer":"🚻 Author",
+    "date":"date",
+    "publication_time":"publish date",
+    "err_active_in_the_chat":"🔴 the Bot is already active in this chat",
+    "active_bot":"Bot activated ✅",
+    "admin_activate":"⛔ You need to be an Admin to activate this Bot",
+    "category_id":"Send the Category's 🆔 you want to subscribe to",
+    "category_id_all":"If you want to receive posts from all categories type '0'",
+    "id":"Identifier",
+    "err_wrong_entry":"Wrong entry",
+    "send_category_id":"Send the Category's 🆔",
+    "topic_title":"🔤 Topic title:",
+    "topic_content":"🔤 Topic content:",
+    "send_me_private_message_to_link_your_account":"🔶 To link your account, send me a private message",
+    "err_linked_to_discourse":"⛔ This account is already Linked to a Discourse account",
+    "enter_your_username_discourse":"Write your Discourse username",
+    "sign_":"username without @",
+    "verification_code":"🔵 verification code",
+    "verification_code_for":"'s verification code",
+    "send_verification_code":"Your verification code has been sent ✅, you can find it in your Discourse DMs",
+    "write_verification_code":"🔵 send the verification code, to enable bridging",
+    "active_bridge":"Bridge Enabled ✅",
+    "err_verification_code":"⛔ Can't verify, is the code correct?",
+    "send_id_or_url_topic":"send the Topics ID or a direct link",
```

`Topics` -> `topics`
```suggestion
    "send_id_or_url_topic":"send the topics ID or a direct link",
```